dev_tools/write-ci-requirements.py

```diff
@@ -0,0 +1,107 @@
+import pathlib
+import re
+import sys
+from argparse import ArgumentParser
+from typing import Any, Dict, List
+
+REPO_DIR = pathlib.Path(__file__).parent.parent.resolve()
+print('Repo dir:', REPO_DIR)
+
+CIRQ_VERSIONS = {
+    'previous': '==0.12.0',
+    'current': '==0.13.0',
+    'next': '>=0.14.0.dev',
+}
```

Comment:
Should we put a note in the requirements.txt file so that people know to go and update this?

Reply:
good idea, done